tests/helper/helper_dev.go

```diff
@@ -105,10 +122,21 @@ func (o DevSession) Stop() {
 	o.session.Interrupt()
 }
 
-// RunDevMode runs a dev session and executes the `inside` code
-func RunDevMode(inside func(session *gexec.Session)) {
-	session := StartDevMode()
+// RunDevMode runs a dev session and executes the `inside` code when the dev mode is completely started
+// The inside handler is passed the internal session pointer, the contents of the standard and error outputs,
+// and a slice of strings - urls - matching the LOCALHOST_REGEXP at the time the handler is called
```

Comment:
```suggestion
// and a slice of strings - urls - giving the redirections in the form localhost:<port_number> to access ports opened by component
```

The regex is clearly an implementation details, the caller don't need to know about this, but wants to know what are these strings about